src/solver/type_getters.jl

```diff
@@ -183,17 +164,27 @@ function get_spaces(parsed_args, params, comms_ctx)
         else
             Meshes.Uniform()
         end
-        if warp_function == nothing
-            make_hybrid_spaces(h_space, z_max, z_elem, z_stretch; deep)
-        else
+        if topography == "Earth"
```

Comment:
```suggestion
        if warp_function == nothing
```

Reply:
(It seems this and the following branches are returning the same thing. They both return `make_hybrid_spaces` with `surface_warp=nothing`.)